faq.mdx

```diff
@@ -66,23 +74,25 @@ OpenTofu will work with existing state files up to those created with Terraform
 
 <AccordionItem id="providers" summary="Does OpenTofu work with all the providers Terraform works with?">
 
-OpenTofu will not have its own providers. Terraform providers have not altered their licenses, and the potential for such a change is virtually zero. OpenTofu works with the current Terraform providers, but it uses a separate registry.
+The [Terraform Plugin SDK v2](https://github.com/hashicorp/terraform-plugin-sdk) and the newer [Terraform Plugin Framework](https://github.com/hashicorp/terraform-plugin-framework) are both available under the [Mozilla Public License 2.0](https://spdx.org/licenses/MPL-2.0.html) (MPL-2.0), an [open-source license](https://opensource.org/licenses?ls=MPL-2.0) ([definition](https://opensource.org/osd)). This means that we can continue leveraging the existing providers and modules in OpenTofu without having to fork them.
+
+Terraform providers have not altered their licenses, and the potential for such a change is virtually zero. OpenTofu works with the current Terraform providers, but it uses a [separate registry](https://search.opentofu.org). If those circumstances change, we will have to adapt. But we are confident that the OpenTofu and its community will be able to resolve any issues that arise.
 
 </AccordionItem>
 
 <AccordionItem id="decisions" summary="How are new features, bug fixes, and other development decisions made in OpenTofu?">
 
-The core team with its technical lead determine the most important features and bug fixes to work on, while the steering committee makes decisions on big changes.
+The Core team with its Technical Lead determine the most important features and bug fixes to work on, while the _Technical Steering Committee_ (TSC) makes decisions on big changes.
```

Comment:
A link to the [TSC notes](https://github.com/opentofu/opentofu/blob/main/TSC_SUMMARY.md) would be helpful, here.